package.json

```diff
@@ -1,21 +1,49 @@
 {
-    "name": "ronadatabase",
-    "version": "1.0.0",
-    "description": "",
+    "title": "Landing Page",
+    "name": "startbootstrap-landing-page",
```

Comment:
needs update

Reply:
This is from the Bootstrap template that I used for the landing page.

Reply:
Right, they're dummy values. Ideally, we would fill them in with actual values for our project. Not a blocker though